GVFS/GVFS/CommandLine/GVFSVerb.cs

```diff
@@ -401,9 +401,10 @@ protected void ValidatePathParameter(string path)
             GitObjectsHttpRequestor objectRequestor,
             GVFSGitObjects gitObjects,
             GitRepo repo,
-            out string error)
+            out string error,
+            bool ignoreIfRootExists = true)

```

Comment:
This name is confusing, and results in a double-negative check down below with `!ignore`. Rename it to `checkLocalObjectCache`?